lib/contentful_rails/engine.rb

```diff
@@ -23,10 +23,10 @@ class Engine < ::Rails::Engine
     #Iterate through all models which inherit from ContentfulModel::Base
     #and add an entry mapping for them, so calls to the Contentful API return
     #the appropriate classes
-    #If ContentfulModel.configuration.entry_mapping is already configured from
-    #initializer, assuming that is the one to use and not running `add_entry_mapping`
+    #If eager_load_entry_mapping is true, engine assumes entry mapping is set manually by
+    #Contentful.entry_mapping (set in ContentfulRails.contentful_options[:entry_mapping])
```

Comment:
Thanks for clearing up the comment, but still I think it's reversed.

It should be if it is `false` then engine assumes it's being manually overwritten.